app/controllers/oaisys/pmh_controller.rb

```diff
@@ -54,12 +54,32 @@ def list_sets
                                                 resumption_token_provided: resumption_token_provided }
   end
 
-  # TODO: Handle the identifier argument.
   def list_metadata_formats
     parameters = expect_args optional: [:identifier]
 
+    if parameters[:identifier].blank?
+      formats = SUPPORTED_FORMATS
+    else
+      uuid = params[:identifier]
+      uuid.slice! 'oai:era.library.ualberta.ca:'
```

Comment:
Do we need to handle this for other verbs, and if so, can we pull this out of the verb and handle it separately (maybe in `expect_args`?)